download.dd

```diff
@@ -83,7 +83,7 @@ $(DIVC download_channels,
   $(BETA
   $(DIVC download_channel,
     $(H2 $(LNAME2 dmd_beta,DMD Beta $(B_DMDV2)-$(B_SUFFIX)))
-    $(LINK2 changelog/$(B_DMDV2).html, Changelog)
+    $(LINK2 changelog/$(B_DMDV2)_pre.html, Changelog)
```

Comment:
So far we made the beta changelog from 2.075.0_pre.dd to 2.075.0.html, something went wrong to make a _pre.html from a (generated?) _pre.dd changelog instead.